### ISSUE TITLE
Windows 32bits version

### ISSUE THREAD

**Issue by mlanderdahl** [daemondevin/GitHubDesktopPortable#7]
Hi!
Is it possible to have a 32bit version of the executable?
I am trying to run this on a machine which is not very new (runs Win 7 32bits) and additionally I don't have admin rights, so `GitHubDesktopPortable` was my best option although it didn't work.

Thanks!

**Comment by Asvsmadhav** [comment]
Yes its possible to have a 32 bit version but its not available in windows 7 it is stable for windows 10 and macOs version 2.0.8.2.9.5 version but unfortunately the below version is not available so please try in windows 8 to 10 or MacOs if available
